src/components/TransactionSettings/index.tsx

```diff
@@ -142,7 +142,8 @@ export default function TransactionSettings({ placeholderSlippage }: Transaction
     } else {
       try {
         const parsed: number = Math.floor(Number.parseFloat(value) * 60)
-        if (!Number.isInteger(parsed) || parsed < 60 || parsed > 180 * 60) {
+        const threeDaysInSeconds = 60 * 60 * 24 * 3
```

Comment:
better to keep const in the global space to avoid recomputing.

also can use [ms](https://www.npmjs.com/package/ms)

```
ms`3 days` * 1000
```

Reply:
Global space of `index.tsx` or in a constants file?

Reply:
you can leave it in `index.tsx` yeah

Reply:
Prefer ```ms`3 days` * 1000``` for readability. Once this is done and moved into the file's global space, it should be good to go. Thank you!

Reply:
Updated

Reply:
we both got it wrong with `* 1000`, glad you used `/ 1000` instead :)